tests/cupy_tests/core_tests/test_ndarray_copy_and_view.py

```diff
@@ -180,11 +180,10 @@ def test_isinstance_numpy_copy(self, xp, dtype, order):
 
     @unittest.skipUnless(util.ENABLE_SLICE_COPY, 'Special copy disabled')
     def test_isinstance_numpy_copy_wrong_dtype(self):
-        for xp in (numpy, cupy):
-            a = numpy.arange(100, dtype=numpy.float64).reshape(10, 10)
-            b = cupy.empty(a.shape, dtype=numpy.int32)
-            with pytest.raises(ValueError):
-                b[:] = a
+        a = numpy.arange(100, dtype=numpy.float64).reshape(10, 10)
+        b = cupy.empty(a.shape, dtype=numpy.int32)
+        with pytest.raises(ValueError):
+            b[:] = a
```

Comment:
Good catch for fixing this! I am not sure why the CI does not catch this...Perhaps we don't have `ENABLE_SLICE_COPY` set up?

Reply:
This is no functional change; it was just that the same exact test were run twice in a row, since `xp` was not used inside the for loop.

Reply:
Oh I misread the line `b = ...` 😅 Thanks!